.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
**Check linter:**

- Linter should be set up correctly for this repository.
- Linter result should be green. AKA: Linter should report NO errors.
- There should be no custom changes to linter config (as an attempt to cheat linter error reports).